compiler/luci/service/src/Nodes/CircleQuantize.cpp

```diff
@@ -24,4 +27,11 @@ luci::CircleNode *CloneNodeLet<CN::OPQR>::visit(const luci::CircleQuantize *)
   return _graph->nodes()->create<luci::CircleQuantize>();
 }
 
+loco::TensorShape sinf::Algorithm::visit(const luci::CircleQuantize *node)
```

Comment:
plz revise this like #13810

Reply:
I applied this to [compiler/luci/service/src/Nodes/CircleQuantize.cpp](https://github.com/Samsung/ONE/pull/13812/files#diff-a1e42208818934f62a8885f36c22d71cb0933d94eb265380fe0880d1302e9eb2)